sdk/appservice/arm-appservice-rest/swagger/README.md

```diff
@@ -0,0 +1,46 @@
+# Azure AppService TypeScript Protocol Layer
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: "@azure-rest/arm-appservice"
+title: WebSiteManagementClient
+description: App Service Client
+generate-metadata: true
+generate-test: true
+generate-sample: true
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../
+source-code-folder-path: ./src
+input-file:
```

Comment:
can we use readme.md instead of adding all json files?

Reply:
updated.